broker-daemon/broker-rpc/info-service/get-supported-markets.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Gets valid markets from the relayer and populates market information for these markets
+ *
+ * @param {Object} request - request object
+ * @param {Object} request.params
+ * @param {RelayerClient} request.relayer
+ * @param {Logger} request.logger
+ * @param {Engine} request.engines
+ * @param {Map<Orderbook>} request.orderbooks
+
+ * @param {Object} responses
+ * @param {function} responses.EmptyResponse
+ * @return {responses.EmptyResponse}
+ */
+async function getSupportedMarkets ({ params, relayer, logger, engines, orderbooks }, { GetSupportedMarketsResponse }) {
+  const { markets } = await relayer.infoService.getMarkets({})
+
+  let supportedMarkets = []
+  markets.forEach((market) => {
+    if (orderbooks.get(market)) {
+      const [base, counter] = market.split('/')
+      const marketInfo = {
+        id: market,
+        symbol: market,
+        base,
+        counter,
+        active: true,
+        precision: 16
```

Comment:
Thinking about where we want to store this information 🤔

Reply:
@martineehrlich this is probably best set on a per-currency basis, however the value of 16 is something that we (sparkswap) have set.

As a counter-example, coinbase/GDAX uses the following `Bitcoin, Bitcoin Cash, Litecoin and Ethereum values will have 8 decimal points and fiat currencies will have two.